chef/cookbooks/hyperv/recipes/openstack_install_networking-hyperv.rb

```diff
@@ -0,0 +1,26 @@
+raise if not node[:platform_family] == "windows"
```

Comment:
Style/NegatedIf: Favor `unless` over `if` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)<br>Style/Not: Use `!` instead of `not`. (https://github.com/bbatsov/ruby-style-guide#bang-not-not)